rust/lance-encoding/src/lib.rs

```diff
@@ -10,7 +10,11 @@ use lance_core::Result;
 
 pub mod decoder;
 pub mod encoder;
-pub mod encodings;
+pub mod encodings {
+    pub mod logical;
+    pub mod physical;
+    pub mod utils;
+}
```

Comment:
Why this change?

Reply:
For the import from utils.rs
But I'll delete the method based on previous comments, so it won't be necessary any longer

Reply:
In the future you can just add `utils` to `rust/lance-encodings/src/encodings.rs`